src/chat/ChatScreen.js

```diff
@@ -18,11 +18,13 @@ import { canSendToNarrow } from '../utils/narrow';
 import { getLoading } from '../directSelectors';
 import { getFetchingForNarrow } from './fetchingSelectors';
 import { getShownMessagesForNarrow } from './narrowsSelectors';
+import { getAuth } from '../selectors';
 
 type SelectorProps = {|
   fetching: Fetching,
   haveNoMessages: boolean,
   loading: boolean,
+  auth: Auth,
```

Comment:
I think this isn't used, now that the API call is made at a different layer.

Reply:
Removed